src/sdk/service/network-validation-service/network-validation-service.ts

```diff
@@ -0,0 +1,475 @@
+import { Range, NetworkParams } from './network-validation-interface';
+
+/**
+ * Network Validation Service
+ */
+export class NetworkValidationService {
+  private static instance: NetworkValidationService;
+
+  /**
+   * @return {NetworkValidationService} instance of NetworkValidationService class.
+   */
+  static getInstance(): NetworkValidationService {
```

Comment:
there is no reason to use a singleton instance for this service. You should just make the class abstract and all methods static.